vo-frontend-igwn/wip-job-wrappers/simple-bearer-token-job-wrapper.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export BEARER_TOKEN_FILE="$_CONDOR_CREDS/ligo.use"
```

Comment:
```suggestion
[[ -n $BEARER_TOKEN_FILE ]] || export BEARER_TOKEN_FILE="$_CONDOR_CREDS/ligo.use"
```
in case it's already defined?

Reply:
Sounds good to me... shouldn't really hurt either way.